### DISCUSSION TITLE
Full example for the React API / `styled` / `windstitch` / `stitches` syntax

### DISCUSSION THREAD

**Discussion by jlarmstrongiv** [joe-bell/cva#132] [Show and tell]
Followup on https://github.com/joe-bell/cva/discussions/123

Full example https://gist.github.com/jlarmstrongiv/1304d306be154578c265b12bab84e035 that you can copy and paste into your project today 😄

There’s only one `as any` that I haven’t figured out how to get rid of yet (coming soon), but the `styled` api, `as` props, and autocomplete work great 🎉

It’s a relatively small utility (🤏 <100 lines) that makes working in React so much nicer

I’d love to see it considered and added to the main library, despite the project FAQs https://cva.style/docs/faqs#why-dont-you-provide-a-styled-api If so, I’d be glad to make a PR for it 🚀

**Comment by joe-bell** [comment]
> I’d love to see it considered and added to the main library

Sorry, but as I’ve said [previously](https://github.com/joe-bell/cva/discussions/123#discussioncomment-5287574) and in the [FAQs](https://cva.style/docs/faqs#why-dont-you-provide-a-styled-api) this isn’t something I plan to support — I don’t plan on changing that stance as I don’t believe there’s a strong case for it
